src/config/index.ts

```diff
@@ -0,0 +1,36 @@
+export const DOCUMENT_STORE_CREATOR_ROPSTEN = "0x4077534e82C97Be03A07FB10f5c853d2bC7161FB";
+export const DOCUMENT_STORE_CREATOR_MAINNET = "0x0";
+export const PROXY_FACTORY_ROPSTEN = "0xba2501bf20593f156879c17d38b6c245ca65de80";
+export const PROXY_FACTORY_MAINNET = "0x0";
+
+let documentStoreCreatorAddressOverwrite: string;
+let proxyFactoryAddressOverwrite: string;
+
+export const getDocumentStoreCreatorAddress = (networkId: number) => {
+  switch (true) {
+    case !!documentStoreCreatorAddressOverwrite:
```

Comment:
what is this for ?

Reply:
If users want to overwrite the instance for a network we are not in. We are on 1 & 3, if they want to be on rinkeby they can still use this by providing their deployed factories. 

More importantly it's used in our test where each time a non-deterministic blockchain is spun up and we will deploy the factories there first and then overwrite it.